### DISCUSSION TITLE
how to allow column resizing to auto reset?

### DISCUSSION THREAD

**Discussion by wwsskkaa** [TanStack/table#2682] [General]
seems like other hooks has those options
autoResetPage: !skipPageResetRef.current,
   autoResetExpanded: !skipPageResetRef.current,
   autoResetGroupBy: !skipPageResetRef.current,
   autoResetSelectedRows: !skipPageResetRef.current,
   autoResetSortBy: !skipPageResetRef.current,
   autoResetFilters: !skipPageResetRef.current,
   autoResetRowState: !skipPageResetRef.current,

is there any reason why resizedColumns doesn't have such auto Reset option? 
what is the implementation logic then? seems like it never auto resets.

**Comment by gargroh** [comment]
It has, check https://github.com/tannerlinsley/react-table/blob/master/src/plugin-hooks/useResizeColumns.js#L249
